.github/workflows/sycl-post-commit.yml

```diff
@@ -116,9 +97,9 @@ jobs:
       ref: ${{ github.sha }}
       merge_ref: ''
 
-      sycl_toolchain_artifact: ${{ matrix.sycl_toolchain_artifact || 'sycl_linux_sprod_shared' }}
-      sycl_toolchain_archive: ${{ matrix.sycl_toolchain_archive || needs.build-lin.outputs.artifact_archive_name }}
-      sycl_toolchain_decompress_command: ${{ matrix.sycl_toolchain_decompress_command || needs.build-lin.outputs.artifact_decompress_command }}
+      sycl_toolchain_artifact: sycl_linux_sprod_static
```

Comment:
I missed it during previous review round, but "sprod_static" doesn't make any sense. Can we rename to using the default name?

Reply:
fixed in latest commit, thx